src/components/AssociatedGamesCollapse/index.module.scss

```diff
@@ -1,5 +1,5 @@
 .associatedGamesContainer {
-  border: 1px solid var(--color-stroke-01);
+  border: 2px solid var(--color-stroke-01);
   padding: var(--space-sm) var(--space-md);
   border-radius: var(--space-sm);
   background-color: var(--color-neutral-800);
```

Comment:
Now we have specific variables for stroke, background, and fonts if you want to change them- of course, if they match what we have on this page already.